bootstrapvz/common/tasks/grub.py

```diff
@@ -26,11 +26,27 @@ def run(cls, info):
 		from bootstrapvz.common.tools import sed_i
 		grub_def = os.path.join(info.root, 'etc/default/grub')
 		sed_i(grub_def, '^#GRUB_TERMINAL=console', 'GRUB_TERMINAL=console')
-		sed_i(grub_def, '^GRUB_CMDLINE_LINUX_DEFAULT="quiet"',
-		                'GRUB_CMDLINE_LINUX_DEFAULT="console=hvc0"')
-		sed_i(grub_def, '^GRUB_TIMEOUT=[0-9]+', 'GRUB_TIMEOUT=0\n'
-		                                        'GRUB_HIDDEN_TIMEOUT=0\n'
-		                                        'GRUB_HIDDEN_TIMEOUT_QUIET=true')
+		
+		# optionally, enable grub menu
+		if info.manifest.system.get('grub_enable_timeout') is True:
+			# print "grub: enabling menu"
+			sed_i(grub_def, '^GRUB_TIMEOUT=[0-9]+', 'GRUB_TIMEOUT=5')
+		else:
+			sed_i(grub_def, '^GRUB_TIMEOUT=[0-9]+', 'GRUB_TIMEOUT=0\n'
+													'GRUB_HIDDEN_TIMEOUT=0\n'
+													'GRUB_HIDDEN_TIMEOUT_QUIET=true')
```

Comment:
Looks like you aligned those lines with tab instead of spaces. Please take a look at the project's [coding style][1].

[1]: https://github.com/andsens/bootstrap-vz/blob/72699d09c8d18fccb6f434023f33c92ca4ad94ca/CONTRIBUTING.rst#coding-style